public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -1796,6 +1801,40 @@ definitions:
         example: Red Hat
       schema:
         type: object
+  Tag:
+    type: object
+    properties:
+      tag_id:
+        type: string
+        readOnly: true
+        format: uuid
```

Comment:
@bdunne should be integer? Or any kin of validation we would do will fail on this

Reply:
See https://github.com/ManageIQ/topological_inventory-api/commit/3bc29df178e71c7174ed2c3299ec0de0d389cb15#diff-93e2df6f7892ddefd3d8b61839de743fR1412